File: src/main/java/domain/math/FindMax.java
package domain.math;

public class FindMax {
    public static Double findMax(Double[][] values) {
        Double max = 0d;

        for(Double[] array: values) {
            for(Double value: array) {
                if(value > max) {
                    max = value;
                }
            }
        }

        return max;
    }
}

File: src/main/kotlin/domain/math/FindMax.kt
package domain.math

object FindMax {
    fun findMax(values: Array<Array<Double>>): Double {
        var max = 0.0
        for (array in values) {
            for (value in array) {
                if (value > max) max = value
            }
        }
        return max
    }
}
